tests/test_guest.py

```diff
@@ -263,6 +263,21 @@ def test_snp_with_ovmfx64_and_kernel_should_fail(self):
         self.assertEqual(str(c.exception),
                          "Kernel specified but OVMF metadata doesn't include SNP_KERNEL_HASHES section")
 
+    def test_snp_with_sev_snp_ovmf(self):
+        ld = guest.calc_launch_digest(
+                SevMode.SEV_SNP,
+                1,
+                vcpu_types.CPU_SIGS["EPYC-v4"],
+                "tests/fixtures/ovmf_amd_sev_snp.fd",
+                "/dev/null",
+                "/dev/null",
+                "",
```

Comment:
This enables kernel hashes. Do we want this here? If not, we should pass `None, None, None` for these params (maybe we should start using named parameters for this function).